OracleSiebel/OEL/build/multi-stage/build/build-all-multistage

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: Dec, 2019
+# Author: duncan.ford@oracle.com
+# Description: Build the Siebel containers with defined tagging and SSL cert
+#
+# Notes: execute this from the multi-stage/build directory
+#
+# arguments
+#       $1 - version to build - this will be the name used to access the update folder in the installation media location
+#       $2 - repository location
+#       $3 - folder within repository location
+#       $4 - domain for SSL certificate generation
+#
+# The product is three containers with the following naming
+#
+#       $2/$3/sai:$1np
+#       $2/$3/cgw:$1np
+#       $2/$3/ses:$1np
+#
+# e.g. bash build-all-multistage 19.11 registry.local.com:5000 siebel *.siebel.local 2>&1 | tee build.$(date +%F_%R).log
+#
+# This will result in three containers with the following naming
+#
+#       registry.local.com:5000/siebel/sai:19.11np
+#       registry.local.com:5000/siebel/ses:19.11np
+#       registry.local.com:5000/siebel/cgw:19.11np
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+BUILD_DATE=`date -I`
+
+function buildElement {
+        echo ...
+        echo Building $1 element for $2
+        echo ...
+        docker build -f ../$1/dockerfile -t $3/$4/$1:${2}np --build-arg INSTALLSITE=installers --build-arg VERSION=$2 --build-arg BUILD_DATE=${BUILD_DATE} --build-arg HTTP_PROXY --build-arg HTTPS_PROXY --build-arg NO_PROXY --network install ..
+}
+
+echo Beginning build for $1
+date
+set -x
+
+# pull latest oraclelinux container before beginning build
+
+docker pull oraclelinux:latest
```

Comment:
Please switch to `oraclelinux:7-slim` as per the [contribution guide](https://github.com/oracle/docker-images/blob/master/CONTRIBUTING.md). If that's not possible, at least switch to `oraclelinux:7` as `latest` will change to 8 fairly soon.

Reply:
Switched